src/utils/QCCalculator.cpp

```diff
@@ -438,7 +438,46 @@ class TOPPQCCalculator :
     }
     qcmlfile.addRunQualityParameter(base_name, qp);
 
-    
+
+    //---injection times MSn
+    at = QcMLFile::Attachment();
+    at.cvRef = "QC"; ///< cv reference
+    at.cvAcc = "QC:0000018";
+    at.qualityRef = msaq_ref;
+    at.id = base_name + "_ms2inj"; ///< Identifier
+    try
+    {
+      const ControlledVocabulary::CVTerm& term = cv.getTerm(at.cvAcc);
+      at.name = term.name; ///< Name
+    }
+    catch (...)
+    {
+      at.name = "MS2 injection time"; ///< Name
+    }
+
+    at.colTypes.push_back("MS:1000894_[sec]");
+    at.colTypes.push_back("MS:1000927");
+    for (Size i = 0; i < exp.size(); ++i)
+    {
+      if (exp[i].getMSLevel() > 1 && exp[i].getAcquisitionInfo().size() > 0)
+      {
+        for (Size j = 0; j < exp[i].getAcquisitionInfo().size(); ++j)
+        {
+          if (exp[i].getAcquisitionInfo()[j].metaValueExists("MS:1000927"))
+          {
+            std::vector<String> row;
+            row.push_back(exp[i].getRT());
```

Comment:
Do you know why does this work? RT is a double isn't it and should never be implicitly be converted to a String?